src/cli/migrate/nodeVersion/index.ts

```diff
@@ -14,12 +14,45 @@ type SubPatch = (
   replace: string;
 };
 
+type VersionResult = {
+  version: string;
+  err: string | undefined;
+};
+
+type RegistryResponse = {
+  'dist-tags': Record<string, string>;
+};
+
+export const getLatestNode22Types = async (): Promise<VersionResult> => {
+  const FALLBACK_VERSION = '22.9.0';
+  const url = 'https://registry.npmjs.org/@types/node';
+  const headers = {
+    accept:
+      'application/vnd.npm.install-v1+json; q=1.0, application/json; q=0.8, */*',
+  };
+
+  try {
+    const fetchResponse = await fetch(url, { headers });
+    const jsonResponse = (await fetchResponse.json()) as RegistryResponse;
+    const latestNode22 = jsonResponse['dist-tags'].node22;
```

Comment:
Does a node22 dist tag exist..?

Reply:
As it turns out... no 😢

Reply:
Swapped it for your suggestion of
`npm show @types/node@^22 version --json`